src/components/Create/components/pages/ReviewDeploy/components/FundingConfigurationReview/FundingConfigurationReview.tsx

```diff
@@ -1,3 +1,107 @@
+import { t } from '@lingui/macro'
+import { Row } from 'antd'
+import { AllocationSplit } from 'components/Create/components/Allocation'
+import { formatFundingCycle } from 'components/Create/utils/formatFundingCycle'
+import { formatFundingTarget } from 'components/Create/utils/formatFundingTarget'
+import { useAppSelector } from 'hooks/AppSelector'
+import { Split } from 'models/splits'
+import { useCallback, useMemo } from 'react'
+import { useEditingDistributionLimit } from 'redux/hooks/EditingDistributionLimit'
+import { useEditingPayoutSplits } from 'redux/hooks/EditingPayoutSplits'
+import { PayoutsList } from '../../../Payouts/components/PayoutsList'
+import { DescriptionCol } from '../DescriptionCol'
+import { emphasisedTextStyle, flexColumnStyle } from '../styles'
+
 export const FundingConfigurationReview = () => {
-  return <></>
+  const { fundingCycleData } = useAppSelector(state => state.editingV2Project)
+  const [distributionLimit] = useEditingDistributionLimit()
+  const [payoutSplits, setPayoutSplits] = useEditingPayoutSplits()
+
+  const fundingCycles = useMemo(
+    () => (fundingCycleData.duration == '0' ? t`Manual` : t`Automated`),
+    [fundingCycleData.duration],
+  )
+
+  const duration = useMemo(
+    () => formatFundingCycle(fundingCycleData.duration),
+    [fundingCycleData.duration],
+  )
+
+  const fundingTarget = useMemo(
+    () =>
+      formatFundingTarget({
+        distributionLimit: distributionLimit?.amount.toString(),
+        distributionLimitCurrency: distributionLimit?.currency.toString(),
+      }),
+    [distributionLimit?.amount, distributionLimit?.currency],
+  )
+
+  const splitToAllocation = useCallback((split: Split): AllocationSplit => {
+    return {
+      id: `${split.beneficiary}${split.projectId ? `-${split.projectId}` : ''}`,
+      ...split,
+    }
+  }, [])
```

Comment:
**question** does this need to be a `useCallback` if there's no dependencies? Could it instead be defined outside this component's scope?

Reply:
I actually change this further up the stack